string bottle
---
geometry_msgs/Pose lid_pose
bool status